Misc/NEWS.d/next/Library/2025-01-22-16-54-25.gh-issue-129205.FMqrUt.rst

```diff
@@ -0,0 +1 @@
+Add ``os.readinto`` to read into a :ref:`buffer protocol <bufferobjects>` from a file descriptor.
```

Comment:
Please copy this entry to https://docs.python.org/dev/whatsnew/3.14.html#os (Doc/whatsnew/3.14.rst).